d2bs/kolbot/libs/bots/DiabloHelper.js

```diff
@@ -393,8 +480,10 @@ AreaInfoLoop:
 		}
 	}
 
-	Pather.useWaypoint(Config.RandomPrecast ? "random" : 107);
-	Precast.doPrecast(true);
+	if (Config.DiabloHelper.SafePrecast) {
```

Comment:
I dont really see the point of adding an extra variable / setting for this. Besides, if you configure RandomPrecast, its a bit confusing if it suddenly isn't doing this for diablo

Reply:
I did this because I didn't find value in the safe precast and there wasn't a way to not do it. It was just a waste of time, when the char does a precast in the attack.js anyway. I'm sure if a char isn't strong enough, the safe precast is necessary to stay alive, but I wanted to stop my chars doing it as it took like 20seconds and then was repeated in the run anyway. Thoughts?